tutorials/R/Phenocam/phenocam-tools/detecting-foggy-phenocam-images/detecting-foggy-phenocam-images.Rmd

```diff
@@ -95,6 +95,8 @@ In *hazer* we can also extract three basic elements of an RGB image :
 2. Darkness
```

Comment:
I couldn't comment on the line above, but please add yourself as a contributor in the YAML header!